react-native-zcash.podspec

```diff
@@ -21,11 +21,12 @@ Pod::Spec.new do |s|
     "ios/RNZcash.swift",
     "ios/ZCashLightClientKit/**/*.swift"
   s.resource_bundles = {
-    "zcash-mainnet" => "ios/ZCashLightClientKit/Resources/saplingtree-checkpoints/mainnet/*.json",
-    "zcash-testnet" => "ios/ZCashLightClientKit/Resources/saplingtree-checkpoints/testnet/*.json"
+    "zcash-mainnet" => "ios/ZCashLightClientKit/Resources/checkpoints/mainnet/*.json",
+    "zcash-testnet" => "ios/ZCashLightClientKit/Resources/checkpoints/testnet/*.json"
   }
   s.vendored_frameworks = "ios/libzcashlc.xcframework"
 
+  s.dependency "MnemonicSwift", "~> 2.2"
   s.dependency "gRPC-Swift", "~> 1.0"
   s.dependency "SQLite.swift", "~> 0.12"
```

Comment:
My original branch updated to `"gRPC-Swift", "~> 1.8"`. Is this worth including?